SmartDeviceLink/SDLUploadFileOperation.m

```diff
@@ -92,7 +92,7 @@ - (void)sdl_sendFile:(SDLFile *)file mtuSize:(NSUInteger)mtuSize withCompletion:
 
     // Wait for all packets be sent before returning whether or not the upload was a success
     __weak typeof(self) weakself = self;
-    dispatch_group_notify(putFileGroup, dispatch_get_main_queue(), ^{
+    dispatch_group_notify(putFileGroup, [SDLGlobals sharedGlobals].sdlProcessingQueue, ^{
```

Comment:
Isn't this a breaking change, since devs will expect to get the response on the main thread?